service/resource/pod/current.go

```diff
@@ -0,0 +1,23 @@
+package pod
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
+	currentPod, err := toPod(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	fmt.Printf("\n")
+	fmt.Printf("pod resource start: TODO reconciling pod\n")
+	fmt.Printf("%#v\n", currentPod)
+	fmt.Printf("pod resource end: TODO reconciling pod\n")
+	fmt.Printf("\n")
```

Comment:
I am going to implement this here. I saw in the logs that pods having the right label will be reconciled here. One of the next steps is to add the right label to the guest cluster worker pods.